titus-server-runtime/src/main/java/com/netflix/titus/runtime/jobmanager/JobManagerConfiguration.java

```diff
@@ -43,9 +43,16 @@
      * applying the regular request timeout might result in large latency increase for all client requests.
      * Instead we set more aggressive timeout for the task relocation service, at the cost of not including the
      * relocation data in the client response.
-     *o
+     *
      * @return fraction of the GRPC request timeout set for the task relocation service requests
      */
     @DefaultValue("0.1")
     double getRelocationTimeoutCoefficient();
+
+    /**
+     * If set to true, the relocation data cache is used when merging task data with the relocation data.
+     * If set to false, a direct call to the relocation service is made each time for each task.
+     */
+    @DefaultValue("true")
```

Comment:
Based on the description, is false even an option? It would seem like it would overwhelm the relocation service.

Reply:
`false` would revert this to the current implementation. Task relocation queries are done in batches so there is a single request for each `findTasks` client call. By using cache we should improve availability of the relocation data, as if the container TaskRelocation restarts, cached would be used for up to 30sec.

Reply:
`made each time for each task.` -> can you change the wording of this to reflect your reply?